csrc/scheduler/matmul.cpp

```diff
@@ -315,8 +318,11 @@ TensorView* Common::cacheAfter(
   // Reduction axis in the original mma output.
   NVF_ERROR(orig_logical.size() == cache_logical.size());
   for (size_t i : arange(orig_logical.size())) {
-    ValGroup vg = graph_->toGroup(orig_logical[i]);
-    graph_->initializeVal(cache_logical[i], vg);
+    // Due to rFactor in split-k, some ID could be missing in the IdGraph.
+    if (graph_->hasGroup(orig_logical[i])) {
```

Comment:
I feel like this pattern should be a helper function.

Reply:
Added a `addAndMapDomain`